pandas/tests/indexes/datetimes/test_setops.py

```diff
@@ -325,7 +325,7 @@ def test_month_range_union_tz_pytz(self):
     def test_month_range_union_tz_dateutil(self):
         tm._skip_if_windows_python_3()
 
-        from pandas._libs.tslib import _dateutil_gettz as timezone
+        from pandas._libs.tslibs.timezones import dateutil_gettz as timezone
```

Comment:
this is not great renaming

Reply:
I don't disagree.

Reply:
though it might be because of generic usage in the function, maybe ``as get_timezone`` would be better

Reply:
It's in a test file.  Why not just import it without an alias (i.e. just `dateutil_gettz`)?

Reply:
that's fine too (and better).